### ISSUE TITLE
[Wear More Rings] The container ring got unequipped. Trying to fix that.

### ISSUE THREAD

**Issue by Almech-Alfarion** [bcmpinc/StardewHack#95]
Recently updated all my mods after coming back to the game after a couple years. I'm getting this error with the latest version, and I think it's also duplication the ring equipped in the first slot into the first open space in the backpack. The saves I'm loading are both single player and a multiplayer game where I was the host from the last times I played.

SMAPI Log:
https://smapi.io/log/1e89f1a6889f46bfb7c23f9cbecbd558

**Comment by bcmpinc** [comment]
There's a lot of people who run into that issue. My mod detects that something went wrong and tries to fix that. Most likely a different mod that moved the ring, or a different mod that made the ring visible such that the player could move it away. But I've no idea what other mod might be causing this. In any case, I've not been able to reproduce it, so I don't even know if it is something I could fix.

**Comment by bcmpinc** [comment]
So according to the log file, you haven't played with version 6.0 of WMR before, so it tries to migrate your rings from an older version of the mod. Has a previous update caused you to lose your rings in the past? If you save your game, does the problem disappear next time you load your game?

```
16:18:32	INFO	Wear More Rings	Save loaded for John.
16:18:32	ERROR	Wear More Rings	The container ring got unequipped. Trying to fix that.
16:18:32	TRACE	Wear More Rings	Deprecated save data not available.
16:18:32	TRACE	Wear More Rings	Farmer John has old ring chest 0.
16:18:32	TRACE	Wear More Rings	Destroying ring chest with id 0.
16:18:32	TRACE	Wear More Rings	Resetting modifiers for John
```

**Comment by Almech-Alfarion** [comment]
I haven't had any issues with losing rings previously. As I said, it's been a couple years since I played. Digging through my backup files, it looks like the last version of the mod I had installed before this refresh was 5.1! Lol. 

I'd mostly been jumping in and out of the save files to try and make sure my mods as a whole are stable and not causing instability/crashes. I didn't think of saving the character, but a quick "save anywhere" and reload just now seems to have cleared the error.

Thanks for helping clearing that up. Given how old my previous version of the mod was, I'm not sure this error would be replicable anymore.
